elephant/statistics.py

```diff
@@ -524,3 +525,101 @@ def instantaneous_rate(spiketrain, sampling_period, form, sigma, m_idx=None,
                                  units=pq.Hz, t_start=t_start)
 
     return rate
+
+
+def psth(spiketrains, binsize, t_start=None, t_stop=None, output='counts',
+         clip=False):
+    """
+    Peristimulus Time Histogram (PSTH) of a list of :attr:`neo.SpikeTrain`
```

Comment:
Or we change the function name to `time_histogram` and have an alias `psth`. 
Why do you think `t_start` should be required? In the worst case it will be extracted from the maximal `t_start` of the spike train objects. But I am also fine with making it a requirement.